test/distributed/org/apache/cassandra/distributed/Constants.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.distributed;
+
+public final class Constants
+{
+    /**
+     * Property defined in {@link org.apache.cassandra.distributed.api.IInstanceConfig} which references the ID of the
+     * {@link org.apache.cassandra.distributed.api.ICluster}.
+     */
+    public static final String KEY_DTEST_API_CLUSTER_ID = "dtest.api.cluster_id";
+
+    /**
+     * Property used by Instances to determine if checking YAML configuration is required; set to false if validation
+     * of the YAML is not desired.
+     */
+    public static final String KEY_DTEST_API_CONFIG_CHECK = "dtest.api.config.check";
```

Comment:
nit: The constants seems to be best fit in the dtest api repo.

Reply:
I agree, although getting this out as quickly as possible to un-break testing makes sense too (i.e. without waiting for an API release). At least with this class, we have a single thing to move to the API project later...

Reply:
For now I feel its best to stay in-tree but open to revisit later?

Reply:
make sense.

Reply:
+1 to unbreak the tests and follow up on later API release